tests/unit/accounts/test_services.py

```diff
@@ -104,11 +104,44 @@ def test_check_password_updates(self, db_session):
         ]
         assert user.password == "new password"
 
+    def test_create_user(self, db_session):
+        user = UserFactory.build()
+        email = "foo@example.com"
+        service = services.DatabaseUserService(db_session)
+        new_user = service.create_user(username=user.username,
+                                       name=user.name,
+                                       password=user.password,
+                                       email=email)
+        user_from_db = service.get_user(new_user.id)
+        db_session.flush()
```

Comment:
I think we do. Since get_user requires a userid, which doesn't exist in the session until it's flushed. I figured the correct behavior to test for was a user gets written to the database, which we don't know until the session is actually flushed in some fashion. 

conversely, I could add a test to check if the new_user returned is added to the current session. That would effectively validate the desired behavior from create_user:

    assert Session.object_session(new_user) is db_session